client/src/components/employer-component/index.jsx

```diff
@@ -0,0 +1,139 @@
+// import * as React from "react";
+
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import { useNavigate, useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+
+import PropTypes from "prop-types";
+
+import { getEmployer, getEmployerContacts } from "../../utils/api";
+
+import UserType from "../../prop-types/UserType";
+import {
+  EmployerContainer,
+  ArrowContainer,
+  MainContainer,
+  Divider,
+} from "./index.styles";
+
+import EmployerInfoComponent from "./employer-info-component";
+import EmployerInformationCard from "./employer-information-card";
+import ContactsInformationCard from "./contacts-information-card";
+
+function EmployerComponent({ getUserById, managedUsers, setSnackBarMessage }) {
+  const navigate = useNavigate();
+
+  // default, this gets overriden once the useEffect() finishes. to avoid a not available error.
+  const [employer, setEmployer] = useState({ owner: 1, creator: 1 });
+  const [contacts, setContacts] = useState({});
+
+  const { employerID } = useParams();
+
+  useEffect(() => {
+    const updateEmployer = async () => {
+      const res = await getEmployer(employerID);
+      const json = await res.json();
+      if (json.status === "success") {
+        const data = json.data.employer;
+
+        await setEmployer(data);
+      }
+    };
+
+    const updateContacts = async () => {
+      const res = await getEmployerContacts(employerID);
+
+      await setContacts(res);
+    };
+
+    updateEmployer();
+    updateContacts();
+  }, []);
+
+  return (
+    <div
+      style={{
+        marginTop: "20px",
+        width: "100%",
+      }}
+    >
+      <div
+        style={{
+          display: "flex",
+          marginRight: "40px",
+        }}
+      >
+        <ArrowContainer>
```

Comment:
We can have the arrow contained within the same header as the Employer name and owner/creator card.

Take a look at edit-job-lead-header to see the set up

Reply:
Fixed! :)